source/_integrations/mastodon.markdown

```diff
@@ -26,6 +26,25 @@ If you want to grant only required accesses, uncheck all checkboxes then check o
 
 {% include integrations/config_flow.md %}
 
+{% configuration_basic %}
+URL:
+  description: The URL of your Mastodon instance, for example `https://mastodon.social`.
+  required: true
+  type: string
+Client key:
+  description: The client key for the application created within your Mastodon account web interface.
+  required: true
+  type: string
+Client secret:
+  description: The client secret for the application created within your Mastodon account web interface.
+  required: true
+  type: string
+Access token:
+  description: The access token for the application created within your Mastodon account web interface.
+  required: true
+  type: string
```

Comment:
```suggestion
```

Reply:
Trust me to copy one that has them (Cambridge Audio and a few others)
I'll fix.